packages/web-components/fast-foundation/src/menu/menu.ts

```diff
@@ -51,7 +53,7 @@ export class Menu extends FoundationElement {
      */
     public connectedCallback(): void {
         super.connectedCallback();
-        this.menuItems = this.domChildren();
+        this.setItems();
```

Comment:
I think we're going to see a subsequent call here with `itemsChanged` as it's going to run when connected as well since that is a change - is there a way to minimize the amount of code that runs twice?

Reply:
Updated.  Now `setItems` won't run on slot contents changed until after it has been run on connectedCallback first.